### ISSUE TITLE
Xaml Islands: Input Pane hides underlying contents for accessibility

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/microsoft-ui-xaml#2182]
**Describe the bug**
When using NVDA, hovering over the XamlIsland reports "DesktopWindowXamlSource" (the AutomationProperty.Name of the input pane).

**Steps to reproduce the bug**
1. Install NVDA
2. Use it
3. Launch PerceptionSimulation or Windows Terminal
4. Hover mouse over the app

**Expected behavior**
NVDA should be able to recognize the control you are hovering on, and identify it for the user.

**Version Info**

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              | Untested   |
| November 2019 Update (18363)  | Yes   |
| May 2019 Update (18362)            | Yes   |
| October 2018 Update (17763)        | Untested   |
| April 2018 Update (17134)          | No Islands   |
| Fall Creators Update (16299)       | No Islands   |
| Creators Update (15063)            | No Islands   |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | Yes   |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Additional context**
If your focus moves to a control inside the Island, then ask NVDA to report the currently focused control, it correctly reports the control. Really, this issue only appears on hover.

**Comment by carlos-zamora** [comment]
This is still an issue

**Comment by LuisSanchez-Dev** [comment]
Still an issue, my laptop came with Windows 11 and I am not able to hover the mouse over these elements.

**Comment by Saharsh979** [comment]
@carlos-zamora Apologies for the confusion. This was closed through our internal ADO-to-GitHub mirroring pipeline. I did not personally make the underlying disposition. The internal decision appears to have been based on prioritization rather than the accessibility report being invalid. I’m coordinating internally to get clarity on the rationale behind this closure and will share additional context once available.
